app/Http/Controllers/App/PostController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Http\Controllers\App;
+
+use App\Http\Controllers\Controller;
+use App\Models\Banner;
+use App\Models\Challenge;
+use App\Models\Comment;
+use App\Models\Post;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use Inertia\Inertia;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Inertia\Response
+     */
+    public function index(Request $request)
+    {
+        $posts = Post::with(['author', 'team', 'banner', 'challenge', 'comments', 'comments.author', 'users'])->orderByDesc('updated_at')->get();
+        $posts = $posts->each(function($post){$post->team->image = ($post->team->image) ? Storage::disk('upload')->url($post->team->image) : null;});
+
+        $user = $request->user();
+        $teams = $user->teams;
+        $teams->load('banner');
+
+        return Inertia::render('post/Index', [
+            'posts' => $posts,
+            'banners' => Banner::all()->keyBy('id'),
```

Comment:
I think we should limit to the banners of the chosen team (via `team.handovers.banner`).

I don't see a case, where I should make a new post regarding another banner than mine.

Reply:
I thought of something like "today I went for a walk and saw the banner hanging from that balcony in my neighborhood". But you are right, that's wuite unlikely. We could restrict it. Or at least preselect the teams banner.

Reply:
I left it in the controller but restricted to the teams banner in the frontend